docs/topics/db/instrumentation.txt

```diff
@@ -94,12 +94,12 @@ To use this, you would create a logger object and install it as a wrapper::
     # Now we can print the log.
     print(ql.queries)
 
-.. currentmodule:: django.db.backends.base.DatabaseWrapper
+.. currentmodule:: django.db.backends.base.base
```

Comment:
This seems inconsistent with `.. currentmodule:: django.core.cache`.

Does `.. currentmodule:: django.db` with `.. method:: execute_wrapper(wrapper)` work here?